timed/employment/serializers.py

```diff
@@ -18,9 +21,12 @@
 from timed.employment import models
 from timed.tracking.models import Absence, Report
 
+if TYPE_CHECKING:
+    from typing import ClassVar
+
```

Comment:
Why is this needed? That looks like an ugly workaround... but for what?

Reply:
Its not evaluated at runtime

Reply:
this is especially useful to circumvent circular imports when typing things, in this case its not really necessary

Reply:
Typing allows referencing other types by string representation, which I prefer instead of this weird dance (That isn't even neccessary here)

Reply:
AFAIK, when using string references you would still import things in a `TYPE_CHECKING` block

Reply:
This still breaks and goes against all intuition, as `ClassVar` is referenced just below. In my own tests, this breaks even if `not TYPE_CHECKING`